.travis.yml

```diff
@@ -0,0 +1,9 @@
+dist: trusty
+language: ruby
+rvm:
+- 2.0
+- 2.1
+- 2.2
+- 2.3
+- 2.4
```

Comment:
I wasn't sure what versions you wanted to officially support so I just grabbed a few to see what would pass.

Reply:
I have no particular plans of what versions of Ruby to support. When I wrote the bulk of this code, back in 2010, the big choice was between 1.8 and 1.9, both of which are now lost in the mists of history.

I see that v2.4 (specifically 2.4.2) is the most recent release at https://www.ruby-lang.org/en/downloads/ -- so testing all the 2.x minor versions seems like a good way to go. Is there any particular reason you omitted 2.0?

Reply:
D'oh! OK, good -- then I think we can be satisfied with how this file is at the moment. Thanks.